README.md

```diff
@@ -22,8 +22,20 @@ Plugin for viewing streams located on the ant-media server. There is also a func
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 ## Installation
 
-```sh
+```bash
 npm install --save @antmedia/videojs-webrtc-plugin
```

Comment:
Hi Yash,
This comment is for integrating this package to a project. So it should be there.